### ISSUE TITLE
forum error

### ISSUE THREAD

**Issue by wwj718** [regisb/openedx-docker#47]
## Steps to reproduce

`/courses/course-v1:test+test+test/discussion/forum/`

## Additional info
I use the latest version： `regis/openedx:hawthorn`

**Comment by regisb** [comment]
Hi @wwj718, thanks for taking the time to signal this issue! It was a very real issue that affected all forums. It should be fixed in the next version of the docker image, which will be built in a few minutes.

**Comment by wwj718** [comment]
@regisb 👍

**Comment by tony-h** [comment]
I upgraded to regis/openedx:hawthorn, but I am still getting the forum errors. The error is present on the demo course and on a new course that we created after upgrading to Hawthorn.

The log reports the 500 error but offers no insights about the problem.
`"GET /courses/course-v1:edX+DemoX+Demo_Course/discussion/forum/ HTTP/1.0" 500 11614`

Things seem to work after the upgrade, but perhaps I didn't perform the update correctly. Here are the steps I took:
```
make stop
git pull origin master
make update
make migrate
make assets
make daemon
```
If someone confirms that the forums work using a clean build, then I'll rebuild from scratch.
